$OpenBSD: patch-src_speech_h,v 1.3 2009/04/16 16:39:17 ajacoutot Exp $

- Don't define USE_ASYNC, this required sem_timedwait() to be present.

--- src/speech.h.orig	Thu Apr 16 18:19:43 2009
+++ src/speech.h	Thu Apr 16 18:20:03 2009
@@ -49,7 +49,7 @@
 
 // will look for espeak_data directory here, and also in user's home directory
 #ifndef PATH_ESPEAK_DATA
-   #define PATH_ESPEAK_DATA  "/usr/share/espeak-data"
+   #define PATH_ESPEAK_DATA  "${PREFIX}/share/espeak-data"
 #endif
 
 typedef unsigned short USHORT;
